src/app/core/layouts/header/header.component.html

```diff
@@ -1,4 +1,29 @@
-<nav class="navbar">
+<header>
+  <div class="navbar">
+    <div class="navbar-left">
+      <a [routerLink]="ESSENTIAL_ROUTES.home.url">{{ ESSENTIAL_ROUTES.home.label }}</a>
+    </div>
+
+    <p-menubar [model]="menuItems()">
+      <ng-template pTemplate="item" let-item>
+        <ng-container *ngIf="!item.command">
+          <a class="p-menuitem-link" [routerLink]="item.routerLink">{{ item.label | translate }}</a>
+        </ng-container>
+        <ng-container *ngIf="item.command">
+          <a class="p-menuitem-link" (click)="item.command()">{{ item.label | translate }}</a>
+        </ng-container>
+      </ng-template>
+      <ng-template pTemplate="end">
+        <app-country-dropdown></app-country-dropdown>
+      </ng-template>
+    </p-menubar>
+  </div>
+</header>
+
+
+
+
```

Comment:
Remove all the comments.

Remove also empty lines (leave only one empty at the end.